src/shared/components/mutationTable/column/ExonColumnFormatter.tsx

```diff
@@ -41,11 +42,12 @@ export default class ExonColumnFormatter {
                 return exonData;
             }
             else {
-                return <span style = {{display:"inline-block", float:"right"}}>            
-                    <span style = {{float:"left",width:"24px", textAlign:"right"}}> {exonData.split("/")[0]} </span>  
-                    <span style = {{float:"left", marginLeft:"4px", marginRight:"4px"}}>/</span>
-                    <span style = {{float:"left",width:"24px", textAlign:"left"}}> {exonData.split("/")[1]} </span>
-                </span>
+                let displayStyle = showAllExon?"inline":"none";
```

Comment:
Although this works I personally think it's a bit hard to read it this way. Cause you have to read the HTML to figure out which things are not being displayed. I think it's easier to read to do something like (not 100% sure if this works, :

```
class Example extends Component {
  render() {
    const showTotalNrOfExons = false;
    return (
      <span>
      5
       {showTotalNrOfExons && (
         <span> / 5</span>
       )}
      </span>
    )
  }
}
```
see e.g.: https://codesandbox.io/s/ovwl49xw05

Reply:
Makes sense?